### PR TITLE
Add proposed enhancement for index integer overflow

### PR DESCRIPTION
**Pull request by lokidoki102**
Added proposed enhancement for index integer overflow, the motivation, and the possible implementation to solve it.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1370,3 +1370,15 @@ In the current implementation, HDBs and Condominiums are allowed to share postal
 
 #### B.9.2 Implementation
 - To enhance the user experience, one potential improvement could be displaying the seller's personal and house details after an "add house" or "delete house" command is performed.
+
+### B.10 Fixing Index Error Message
+
+#### B.10.1 Motivation
+- In the current implementation, the commands that uses `INDEX`, such as `editBuyer`, `editSeller`, `delete` and `view`,
+shows a wrong error message when the `INDEX` exceeds the maximum int value. It would show an error message for negative
```
Comment (KhoonSun47) [inline docs/DeveloperGuide.md:1378]: Wrong grammar.

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1370,3 +1370,15 @@ In the current implementation, HDBs and Condominiums are allowed to share postal
 
 #### B.9.2 Implementation
 - To enhance the user experience, one potential improvement could be displaying the seller's personal and house details after an "add house" or "delete house" command is performed.
+
+### B.10 Fixing Index Error Message
```
Comment (felixchanyy) [inline docs/DeveloperGuide.md:1374]: Perhaps should change to Fix

**Review by KhoonSun47 — APPROVED** [review]
LGTM!!!!! Lets goooo

**Review by felixchanyy — APPROVED** [review]
LGTM

**Pull request merged**